services/portal/dataUpload/dataUploadQuestions.js

```diff
@@ -0,0 +1,37 @@
+let chai = require('chai');
+let expect = chai.expect;
+const I = actor();
+
+const dataUploadProps = require('./dataUploadProps.js');
+const portal = require('../../../utils/portal.js');
+const util = require('util');
+
+/**
+ * dataUpload Questions
+ */
+module.exports = {
+  async isNumberAndSizeOfUnmappedFilesCorrect(count, size) {
+    const expectString = util.format(dataUploadProps.unmappedFilesStringFormat, count, size);
+    I.waitForText(expectString, 5);
+  },
+
+  canSeeAllUnmappedFilesOnPage(unmappedFiles) {
+    for (let i = 0; i < unmappedFiles.length; i ++) {
+      I.waitForText(unmappedFiles[i], 5);
+    }
+  },
+
+  async isSuccessfullySubmitted() {
+    const result = await I.grabTextFrom('.map-files__notification-wrapper');
```

Comment:
correct me if i'm wrong, but i thought the `notification-wrapper` contained the result of the submission both in case of success and failure. so here don't you need to check what's in `result` to know if the submission succeeded?